Examples/JavaScript/JavaScript Essentials/Hello World.ts

```diff
@@ -0,0 +1,34 @@
+//// { order: 0, compiler: { target: "es5" } }
+
+// Welcome to the TypeScript playground, this site is a lot like
+// running a TypeScript project inside your editor but inside a
+// web browser.
+
+// The playground makes it easy for you to safely experiment with 
+// ideas in TypeScript by making it trivial to share these projects
+// because the URL for this page is everything required to load 
+// the project for someone else.
+
+const hello = "Hello"
+
+// You can see on the right the result of the TypeScript compiler:
+// this is vanilla JavaScript which can run on browsers, servers
+// or anywhere really.
+
+const world = "World"
+
+// You can see how it makes tiny changes to the code, by converting
+// a "const" to a "var". This is one of the many things TypeScript
```

Comment:
it's not doing this right now

Reply:
The compiler settings aren't being set yet - it will 👍